package.json

```diff
@@ -56,10 +68,12 @@
     "@babel/core": "^7.25.2",
     "@eslint/eslintrc": "^3.2.0",
     "@eslint/js": "^9.16.0",
-    "@medplum/fhirtypes": "^3.2.24",
+    "@medplum/definitions": "^3.2.25",
+    "@medplum/fhirtypes": "^3.2.25",
+    "@medplum/mock": "^3.2.25",
+    "@testing-library/react-native": "^12.9.0",
```

Comment:
Those are under dev dependencies. Perhaps it's the right place?

Reply:
From these, I'd say `@medplum/fhirtypes` could be placed under `"dependencies"` because it's often imported in app files.